onnxruntime/contrib_ops/cpu/signal/dft.h

```diff
@@ -8,16 +8,20 @@ namespace contrib {
 
 class DFT final : public OpKernel {
   bool is_onesided_ = true;
+  int64_t axis_ = 0;
  public:
   explicit DFT(const OpKernelInfo& info) : OpKernel(info) {
     is_onesided_ = static_cast<bool>(info.GetAttrOrDefault<int64_t>("onesided", 0));
+    axis_ = static_cast<bool>(info.GetAttrOrDefault<int64_t>("axis", 0));
```

Comment:
>< [](http://example.com/codeflow?start=23&length=1)

I think the static_cast needs to be removed :)

Reply:
yes!